.github/workflows/ci.yaml

```diff
@@ -59,6 +59,9 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install tox sphinx
 
+      - name: Add problem matcher
```

Comment:
```suggestion
      - name: Add sphinx problem matcher
```

Reply:
I thought it was clear enough that the problem matcher was for the current job (which is just `test-docs` here), but I'm happy enough to make this change. I guess we'd want it changing for flake8 too?

Reply:
That's a good point, I don't really care either way tbh